### ISSUE TITLE
Still no consensus node 

### ISSUE THREAD

**Issue by jagg266** [pi-node/instructions#205]
![Screenshot (4)](https://user-images.githubusercontent.com/94882684/142966950-419b56a9-30d9-4b16-be33-a6f4b64ea645.png)

**Comment by justin6652** [comment]
consensus manually enable by pi core team! i hang up 3 month also dont get it~still waiting

**Comment by jagg266** [comment]
Damn okay I was hoping after 90% uptime they send the packets and where and
how do you download them or is it automatically updated

**Comment by justin6652** [comment]
automatically enable by pi network core team

**Comment by smith111707** [comment]
The same here.
Seems, they just kidding us.
Nothing works, no solutions and what's really killing me - **no answers**. Seems, they just earning money by showing the ad in the app and laughing about how stupid we are trusting them and keeping clicking the damn f'ing button...

**Comment by jagg266** [comment]
I really hope they don’t pull that id be so mad

**Comment by Szaboadrii22** [comment]
Nagyon

**Comment by Szaboadrii22** [comment]
Hell

**Comment by Szaboadrii22** [comment]
Nem

**Comment by Szaboadrii22** [comment]
Legjobb

**Comment by Szaboadrii22** [comment]
Legjobb

**Comment by yeahthatscool** [comment]
I'm having the same issue , tried multiple times and still not enabled. Pii Team, please help us, we are on the same team here.

**Comment by jagg266** [comment]
I’m now at 96% up time and no packets sent to me yet

**Comment by cobelltx** [comment]
Same here... I was trying to figure out what I did wrong as I completed everything or so it said -> Node Installation date:  November 10, 2021 - availability 97.79%. Consensus contain still says absent and consensus container enabled still says no. Of course, everything under Consensus says N/A or No. Anyone have any luck getting assistance from PI support for help and resolution?

![image](https://user-images.githubusercontent.com/97173197/148229494-a0b98b92-9ad5-4202-b67f-1735d5a39379.png)

**Comment by ihatejam** [comment]
This is _normal_ for now until the Core Team start selecting more nodes to participate on the blockchain. Please leave it running in this state as it will still be accumulating "reliability time" which will accumulate rewards per the new mining formula.

**Comment by yeahthatscool** [comment]
Yeah same here mine has been running for over 90 days at the same rate. Hopefully they will add our nodes soon.
